pkg/platform/kube/functionres/lazy.go

```diff
@@ -998,6 +1033,74 @@ func (lc *lazyClient) createOrUpdateIngress(functionLabels labels.Set,
 	return resource.(*ext_v1beta1.Ingress), err
 }
 
+func (lc *lazyClient) deleteExistingCronJobs(functionName, functionNamespace string) error {
+	return lc.kubeClientSet.BatchV1beta1().
+		CronJobs(functionNamespace).
+		DeleteCollection(&meta_v1.DeleteOptions{},
+			meta_v1.ListOptions{LabelSelector: fmt.Sprintf("nuclio.io/function-name=%s", functionName)})
+}
+
+func (lc *lazyClient) createOrUpdateCronJob(functionLabels labels.Set,
```

Comment:
imo this should not handle _only_ processor cron trigger cron jobs, but all kind of cron jobs so we would have to decouple between _function cron jobs_ and its  _cron trigger_  implementation